openadapt/adapters/ultralytics.py

```diff
@@ -82,22 +74,16 @@ def do_fastsam(
     conf: float = 0.4,
     # discards all overlapping boxes with IoU > iou_threshold
     iou: float = 0.9,
-    max_retries : int =  5,
-    retry_delay_seconds: float = 0.1,  
-    
+    max_retries: int = 5,
+    retry_delay_seconds: float = 0.1,
 ) -> Image:
     model = FastSAM(model_name)
 
     imgsz = imgsz or image.size
 
     # Run inference on image
     everything_results = model(
-        image,
-        device=device,
-        retina_masks=retina_masks,
-        imgsz=imgsz,
-        conf=conf,
-        iou=iou,
+        image, device=device, retina_masks=retina_masks, imgsz=imgsz, conf=conf, iou=iou
```

Comment:
This should not be necessary. Please make sure you are using the recommended black/flake8 configs